Modules/_uuidmodule.c

```diff
@@ -91,45 +111,1649 @@ py_windows_has_stable_node(void)
 #endif /* MS_WINDOWS */
 
 
-static int
-uuid_exec(PyObject *module)
-{
-#define ADD_INT(NAME, VALUE)                                        \
-    do {                                                            \
-        if (PyModule_AddIntConstant(module, (NAME), (VALUE)) < 0) { \
-           return -1;                                               \
-        }                                                           \
-    } while (0)
+typedef struct uuidobject {
+    PyObject_HEAD
+    uint8_t bytes[16];
+    Py_hash_t cached_hash;
+    PyObject *is_safe;
+    PyObject *weakreflist;
+} uuidobject;
 
-    assert(sizeof(uuid_t) == 16);
-#if defined(MS_WINDOWS)
-    ADD_INT("has_uuid_generate_time_safe", 0);
-#elif defined(HAVE_UUID_GENERATE_TIME_SAFE)
-    ADD_INT("has_uuid_generate_time_safe", 1);
-#else
-    ADD_INT("has_uuid_generate_time_safe", 0);
-#endif
 
-#if defined(MS_WINDOWS)
-    ADD_INT("has_stable_extractable_node", py_windows_has_stable_node());
-#elif defined(HAVE_UUID_GENERATE_TIME_SAFE_STABLE_MAC)
-    ADD_INT("has_stable_extractable_node", 1);
+// UUID Structure per RFC 9562:
+//
+// A UUID is 128 bits (16 bytes) represented as:
+//
+// String:       xx xx xx xx - xx xx - Mx xx - Nx xx - xx xx xx xx xx xx
+// Byte pos:     0  1  2  3    4  5    6  7    8  9    10 11 12 13 14 15
```

Comment:
Time information has no meaning outside v1 and v6

Reply:
Also 7?

I'll amend the comment

Reply:
done

Reply:
In v7, the time information is different as it has a different granularity. v1 and v6 are identical in the sense that only fields are reordered